### PR TITLE
Bug fix in login/logout template.  

### PR DESCRIPTION
**Pull request by asiyani**
#### Pre-Submission Checklist
- [x] Your pull request targets the `develop` branch.
- [ ] Branch starts with either `fix/` or `feature/`
- [ ] All new and existing tests pass the command `npm test`.
- [ ] Any new files are included in the grunt tasks

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:
- [x] Tested changes locally.
- [ ] New tests added?
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Full Description of your Changes
Removed isloggedin variable from app.js and added req.isauthenticated()
This will check all request sent to server. 
Front end Changed fetch to XMLHttpRequest. 
removed profile from menu on profile page and added home.

### CODE REVIEW

File: views/authenticated.ejs [hunk 1]
```diff
@@ -18,9 +18,9 @@ With ESFiddle you can sample ES6 (ECMAScript 2015) and ES7 (ECMAScript 2016) wit
     <div class="header">
       <nav>
         <div class="logo"><a href="/"><img src="/images/logo.png"/></a></div>
+        <span class="loginBtns"><a href="/">Home</a></span>
         <span class="loginBtns only-loggedout"><a href="/auth/github">Login</a></span>
         <span class="loginBtns only-logged"><a href="/github/logout">Logout</a></span>
-        <span class="loginBtns only-logged-profile"><a href="/github/myProfile">Profile</a></span>
```
Comment (SaraVieira) [inline views/authenticated.ejs]: Why did you remove profile here ?

Comment (asiyani) [inline views/authenticated.ejs]: becz this page logs when you are in myProfile page already.

File: src/authenticated.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
         onlyLogged = document.querySelector('.only-logged'),
-        onlyLoggedProfile = document.querySelector('.only-logged-profile');
+        onlyLoggedProfile = document.querySelector('.only-logged-profile'),
+        authReq = new XMLHttpRequest();
 
     // See if user is authenticated
-    fetch('/authenticated')
-    .then((resp) => resp.json())
-    .then((data) => {
+    authReq.open('GET', '/authenticated', true);
```
Comment (SaraVieira) [inline src/authenticated.js]: Why did you remove fetch here? It's already supported by all major browsers and when 3.0.0 comes out it will also be supported by safari

File: views/authenticated.ejs (hunk 1 above)
Comment (SaraVieira) [inline views/authenticated.ejs]: I don't think we should remove actions from the navbar maybe just remove the link and put a hash tag in it instead of the full path, what do you think?

File: src/authenticated.js (hunk 2 above)
Comment (asiyani) [inline src/authenticated.js]: Old fetch request didnt had cookie in header , without that password.js will send false for authentication.

File: views/authenticated.ejs (hunk 1 above)
Comment (asiyani) [inline views/authenticated.ejs]: ya thats fine as well..

Comment (SaraVieira) [inline views/authenticated.ejs]: Awesome 😄

File: src/authenticated.js (hunk 2 above)
Comment (asiyani) [inline src/authenticated.js]: Yup this code is working fine. 
But i am not sure about using fetch. its not supported in IE and Safari yet and not in mobile browser. [source](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Browser_compatibility)

Comment (SaraVieira) [inline src/authenticated.js]: I can add a polyfill there is a very small one arround and I'll integrate it in the site. 

I'm refactoring the js to divide into modules and I'll do that too

**Comment by SaraVieira** [comment]
Did you make the changes? If so you can merge :)

File: src/authenticated.js (hunk 2 above)
Comment (SaraVieira) [inline src/authenticated.js]: Keep the one you are using here, I'm. Not using the polyfill in this file

**Comment by joshghent** [comment]
Thanks again @asiyani. Looks great!
